### DISCUSSION TITLE
Welcome message and Enable logo

### DISCUSSION THREAD

**Discussion by Whiskey6Actual** [realmazharhussain/gdm-settings#91] [Q&A] [answered]
1) Is there a way to change the size and/or color of the Welcome message?
2) Enable logo isn't working for me. I've tried a couple of different .png files but they don't show up.

I'm running Pop OS 22.04, Gnome 42.2.

**Comment by realmazharhussain** [comment, marked as answer]
> Is there a way to change the size and/or color of the Welcome message?

No.

> Enable logo isn't working for me. I've tried a couple of different .png files but they don't show up.

You can try to run the app from terminal with `-v` or `--verbose` option and if any errors show up, they might indicate what is happening.

Also, sometimes, GDM just refuses to respect some settings as you can see in this bug report (realmazharhussain/gdm-tools#17) for [gdm-tools](https://github.com/realmazharhussain/gdm-tools) which works similar to this app behind the scenes.
